src/utils.js

```diff
@@ -527,3 +515,22 @@ export const parseScriptData = (scriptData, network) => {
     return null;
   }
 }
+
+/**
+ * Call the async wallet method `isAddressMine` considering the type of wallet.
+ *
+ * @param {Object} wallet A wallet instance
+ * @param {string} address A wallet address to check
+ * @param {boolean} useWalletService A flag that determines if wallet service is in use
+ */
+export const isAddressMine = async (wallet, address, useWalletService) => {
+  // XXX: Wallet Service doesn't implement isAddressMine.
+  // See issue: https://github.com/HathorNetwork/hathor-wallet-lib/issues/732
+  // Default to `false` if using Wallet Service.
+  if (useWalletService) {
+    return false;
+  }
```

Comment:
I prefer to throw an error here instead of returning the wrong value/information to the user. We should never use this method with the wallet service until we implement it there.

What do you think?

Reply:
I think we should not do that. The `isMine` flag in transaction history is not a required property, it is a convenience. We are not using the wallet service with nano contract right now, but in case we do, the only consequence will be a list of transactions without a signal if any transaction is our or not. Set the value of `isMine` to `false` for all transactions is not wrong, we can think about it as muting a signaling that is not possible.